src/Helpers/ImageManipulationHelper.ts

```diff
@@ -67,17 +67,26 @@ export const resizeTo = async function (
 export const breakpointSmallerThan = (breakpoint: number, { width, height }: FileDimensions) =>
   breakpoint < width! || breakpoint < height!
 
-export const allowedFormats: Array<AttachmentOptions['forceFormat']> = [
+export type AllowedFormats = 'jpeg' | 'png' | 'webp' | 'avif' | 'tiff' | 'gif' | 'svg' | 'svg+xml'
+
+// Image formats you can upload
+export const allowedFormats: Array<AllowedFormats> = [
   'jpeg',
   'png',
   'webp',
   'avif',
   'tiff',
+  'gif',
+  'svg',
+  'svg+xml',
 ]
 
+// Formats we want to be safely processed by sharp
+export const conversionFormats: Array<AllowedFormats> = ['jpeg', 'png', 'webp', 'avif', 'tiff']
```

Comment:
Rename variable to `processableFormats`. Also append `as const` to the array. 

You should also exclude the non-processable formats from the assigned type: `Array<AllowedFormat>`. E.g. `Array<Exclude<AllowedFormat, 'gif' | 'svg' | 'svg+xml'>>`